temporian/core/operators/arithmetic_scalar/base.py

```diff
@@ -0,0 +1,105 @@
+# Copyright 2021 Google LLC.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Base class for arithmetic scalar operators"""
+
+from abc import abstractmethod
+
+from temporian.core.data.dtype import DType
+from temporian.core.data.dtype import python_type_to_temporian_dtype
+from temporian.core.data.event import Event
+from temporian.core.data.feature import Feature
+from temporian.core.operators.base import Operator
+from temporian.proto import core_pb2 as pb
+
+
+class BaseArithmeticScalarOperator(Operator):
+    """Base Arithmetic scalar operator."""
+
+    def __init__(
+        self,
+        event: Event,
+        value: any,
```

Comment:
this typehint should be `Any` (`any` references the builtin function)

Reply:
also, looking at line 43, we're assuming that `value` is a subclass of either `str`, `int`, `bool`, or `float` (otherwise the code will fail). I think we can improve this type hint